### DISCUSSION TITLE
Many folder merged, new file/folder location

### DISCUSSION THREAD

**Discussion by billouetaudrey** [trapexit/mergerfs#1229] [Q&A]
Hey, I merged successfully my folder with 

sudo mergerfs /mnt/Elements2:/mnt/Elements/:/mnt/3T:/mnt/1To:/mnt/Maxtor/:/mnt/NAS /mnt/Virtual/ -o allow_other

I see all my file on Virtual folder

Now, when I do "touch test" on /mnt/Virtual
I can see it on /mnt/1To

How it's select the hdd ?

Thanks

**Comment by trapexit** [comment]
Policies: https://github.com/trapexit/mergerfs#functions-categories-and-policies

**Reply by billouetaudrey** [reply]
Thanks, my knowledge in English is too poor, will try to translate tomorrow...
Thanks
